### PR TITLE
S2U-44 5.2.8.2 Portal – Visibility of the role that the user is using…

### PR DESCRIPTION
**Pull request by JuanDavid102**
… in a site (#11155) - Master

### CODE REVIEW

**Comment by mpellicer** [comment]
@JuanDavid102 Could you post some screenshots about how it looks like?

File: config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties [hunk 1]
```diff
@@ -120,6 +120,10 @@
 # DEFAULT: false
 # portal.publish.site.confirm.enabled=true
 
+# S2U-44 Display the current user role in the portal.
+# DEFAULT: true
+# portal.display.current.role=false
```
[the remark this answers is not in the document]
Comment (mpellicer) [reply config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties:125]: I meant the default, disabled by default.

**Comment by JuanDavid102** [comment]
How it looks:
imagen
imagen
imagen

**Comment by mpellicer** [comment]
Thank you! It looks great, very good job

**Review by mpellicer — APPROVED** [review]

**Review by jesusmmp — APPROVED** [review]

**Pull request merged**